templates/prebuild.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+##*
+## EDIT: If your packages are CUDA or Python independent, use these 
```

Comment:
Should be dependent, not independent

Reply:
Maybe it should be `version independent`. Version independent then you only want to upload once. Version dependence would be uploading for each version.